requirements-tests.txt

```diff
@@ -1,3 +1,5 @@
 pytest>=7.0.0
 pytest-cov
+pytest-rerunfailures==11.0
```

Comment:
This requirement file is used by tox to prepare virtual environment and install all packages that are required to execute unit tests. Adding other packages which are not needed to run unit tests will increase execution time and usage of disk space.

Reply:
I understand. So, where should we place those packages? Directly to `requirements.txt`, or create new file, like  `requirements-ci.txt`?

Reply:
This package should be rather moved to `requirements.txt`, we need them not to run unit-tests, but to execute scenarios from yaml etc.

Reply:
So, we should place in `requirements.txt` `xdist` package too?

Reply:
`xdist` is used in unit tests so it has to be in `requirements-tests.txt` which is used to install packages by tox
https://github.com/zephyrproject-rtos/twister/blob/3a24a63fe25fad5d5eb88493ce3c72d3a39698bb/tox.ini#L7-L10